groovy/material/FirstDegreeMaterials.groovy

```diff
@@ -3788,7 +3788,40 @@ public class FirstDegreeMaterials {
                 .color(0x2c4a1e)
                 .build()
                 .setFormula("UO2", true);
-
         
+        DiluteNitricAcid = new Material.Builder(8665, SuSyUtility.susyId('dilute_nitric_acid'))
```

Comment:
this has already been added. also, side note: we always use "diluted" not "dilute"